templates/search.html

```diff
@@ -23,7 +23,8 @@ <h1>Found {{ count }} results.</h1>
     <div>
       <h2><a href="/dataset/{{ result.source }}/{{ result.id }}">{{ result.dataset.title }}</a></h2>
 
-      <p>{{ result.dataset.description }}</p>
+
+      <p>{% if let Some(description) = result.dataset.description %} {{ description }} {% else %} (keine Beschreibung) {% endif %}</p>
```

Comment:
I think on the search page, visual space is at a premium and we should just leave whole `<p>` element out if there is no description.